projects/lib/src/dynamic-views/item.component.html

```diff
@@ -1,61 +1,68 @@
-<ng-container *ngIf="config.type === '_container'">
-  <div *ngIf="configurable" class="dragPlaceholder" dndPlaceholderRef>+</div>
+<div
+  *ngIf="config.type === '_raw-html' && condition"
+  [innerHTML]="config.rawHtml">
+</div>
+
+<ng-container
+  *ngIf="!config.type.startsWith('_') && condition"
+  [ngTemplateOutlet]="templates[config.type || id]?.template"
+  [ngTemplateOutletContext]="{ $implicit: config, data: _data, id: id }"
+></ng-container>
+
+<ng-container *ngIf="config.type === '_container' && !configurable">
+  <div *ngFor="let item of config.items; let index = index"
+  [uib-item]="item"
+  [id]="item"
+  [templates]="templates"
+  [zone]="zone"
+
+  [configurable]="false"
+
+  [data]="data"
+  [dataIndex]="dataIndex"
+  [conditionsData]="conditionsData"
+  ></div>
+</ng-container>
+
+<!-- mode editeur -->
+<ng-container *ngIf="config.type === '_container' && configurable">
+  <div
+    [ngClass]="config.classes"
+    class="dropzone"
+
+    style="display: flex;"
 
-  <ng-container *ngIf="condition">
-    <ng-container *ngIf="!configurable">
-      <div #ref *ngFor="let item of config.items; let i = index"
+    [dndDropzone]="[zone]"
+    [dndHorizontal]="isHorizontal()"
+    (dndDrop)="onDndDrop($event)"
+    >
 
+    <div class="dragPlaceholder align-self-stretch" dndPlaceholderRef></div>
+
+    <div *ngFor="let item of config.items; let index = index"
       [uib-item]="item"
       [id]="item"
       [templates]="templates"
-      [configurable]="configurable"
+      [zone]="zone"
+      [configurable]="true"
+
+      [ngClass]="{'dropzone-content': isContainer(item)}"
+
+      [uib-tooltip]="isContainer(item) ? '' : item"
```

Comment:
I propose just `[uib-tooltip]="item"` (seeing the id of the container is useful)

Reply:
If we do this, when we hovering an element, all element downside (hierarchically speaking) will display a tooltip too

Reply:
done. So you can test this behavior